spec.html

```diff
@@ -7600,17 +7600,23 @@ <h1>GetGlobalObject ( )</h1>
   <emu-clause id="sec-jobs" oldids="sec-jobs-and-job-queues,sec-enqueuejob,sec-runjobs,job-queue">
     <h1>Jobs and Host Operations to Enqueue Jobs</h1>
     <p>A <dfn id="job">Job</dfn> is an abstract closure with no parameters that initiates an ECMAScript computation when no other ECMAScript computation is currently in progress.</p>
+    <p>A Job is stored in a <dfn>Job Record</dfn>, which has the slots [[AbstractClosure]], [[Realm]], and [[ScriptOrModule]].</p>
     <p>Jobs are scheduled for execution by ECMAScript host environments. This specification describes the host hook HostEnqueuePromiseJob to schedule one kind of job; host environments may define additional abstract operations which schedule jobs. Such operations accept a Job abstract closure as the parameter and schedule it to be performed at some future time. Their implementations must conform to the following requirements:</p>
 
     <ul>
-      <li>At some future point in time, when there is no running execution context and the execution context stack is empty, the implementation must:
-        <ol>
-          <li>Push an execution context onto the execution context stack.</li>
-          <li>Perform any implementation-defined preparation steps.</li>
-          <li>Call the abstract closure.</li>
-          <li>Perform any implementation-defined cleanup steps.</li>
-          <li>Pop the previously-pushed execution context from the execution context stack.</li>
-        </ol>
+      <li>At some future point in time, when there is no running execution context and the execution context stack is empty, the implementation must perform the following steps:
+        <emu-alg>
+          1. Let _job_ be the selected Job Record.
+          1. Let _context_ be a new ECMAScript code exceution context.
```

Comment:
Orthogonal to whether execution contexts are even needed, it looks like HTML doesn't push a _new_ context every time: https://html.spec.whatwg.org/#prepare-to-run-script

Reply:
i can remove "new" i suppose